doc/topics/packaging/index.rst

```diff
@@ -10,10 +10,72 @@ Relenv onedir packaging
 Starting in 3006, only onedir packaging will be available. The 3006 onedir packages
 are built with the `relenv <https://github.com/saltstack/relative-environment-for-python>`_ tool.
 
+
+How to build onedir only
+========================
+
+Install relenv:
+
+.. code-block:: bash
+
+   # pip install relenv
+
+Fetch toolchain
+
+.. code-block:: bash
+
+   # relenv toolchain fetch --arch=<architecture>
+
+Fetch Native Python Build:
+
+.. code-block:: bash
+
+   # relenv fetch --arch=<architecture>  --python=<pythonversion>
+
+Create relenv environment:
+
+.. code-block:: bash
+
+   # relenv create --arch=x86_64 --python=3.10.10 <relenv name>
+
+Add Salt into onedir. Make sure you are running this in the checkout of Salt:
+
+.. code-block:: bash
+
+   # path/to/relenv name/bin/pip install .
+
+
 How to build rpm packages
 =========================
+Install the dependencies:
 
-You only need to run rpmbuild in the Salt repo:
+.. code-block:: bash
+
+   # yum -y install python3 python3-pip openssl git rpmdevtools rpmlint systemd-units libxcrypt-compat git
```

Comment:
We could even link the custom containers we use to package here: https://github.com/saltstack/salt-ci-containers/tree/main/custom/packaging